app/controllers/buyers/applications_controller.rb

```diff
@@ -191,8 +187,18 @@ def find_provider
     @provider = current_account
   end
 
-  def find_application_plan
-    @application_plan = @provider.application_plans.find params[:cinstance][:plan_id]
+  def accessible_services
+    @accessible_services ||= current_user.accessible_services.includes(:application_plans)
+  end
+
+  helper_method :accessible_services
```

Comment:
a bit weird that `accessible_services` is a helper method and `accessible_plans` not..

Reply:
because accessible_services is used from the view, look:
https://github.com/3scale/porta/blob/c10f20c670c91c4ed63eb43bf601cbc1fe0fd028/app/controllers/buyers/applications_controller.rb#L55
https://github.com/3scale/porta/pull/1798/files#diff-35521f5b3d5085d49b2eac64c87211feR69
https://github.com/3scale/porta/pull/1798#discussion_r409777996